spacialgaze-plugins/SG.js

```diff
@@ -7,16 +7,12 @@ const Autolinker = require('autolinker');
 let regdateCache = {};
 
 SG.nameColor = function (name, bold) {
-	return (bold ? "<b>" : "") + "<font color=" + hashColorWithCustoms(name) + ">" + (Users(name) && Users(name).connected && Users.getExact(name) ? Chat.escapeHTML(Users.getExact(name).name) : Chat.escapeHTML(name)) + "</font>" + (bold ? "</b>" : "");
+	return (bold ? "<b>" : "") + "<font color=" + SG.hashColor(name) + ">" + (Users(name) && Users(name).connected && Users.getExact(name) ? Chat.escapeHTML(Users.getExact(name).name) : Chat.escapeHTML(name)) + "</font>" + (bold ? "</b>" : "");
 };
 // usage: SG.nameColor(user.name, true) for bold OR SG.nameColor(user.name, false) for non-bolded.
 
-SG.hashColor = function (user) {
-	return hashColorWithCustoms(user);
-};
-
 SG.messageSeniorStaff = function (message, pmName, from) {
-	pmName = (pmName ? pmName : '~Upper Staff PM');
+	pmName = (pmName ? pmName : 'SG Server');
```

Comment:
- You forgot the space to specify rank
- This should stay as `~Upper Staff PM` so we can tell when its a upper staff pm.